examples/maskrcnn-example/example_config.ini

```diff
@@ -0,0 +1,32 @@
+[DATA]
+train_data_folder = '/path/to/train/data/folder'
+train_ann = '/path/to/train/annotation/file'
+
+validation_data_folder = '/path/to/validation/data/folder'
+validation_ann = '/path/to/validation/annotation/file'
+
+[LOGGING]
+# calculating mAP takes too much time, so we calculate it in every nth step
+wandb_enabled = True
+wandb_project = wandb_project_name
+performance_tracking_interval = 3
+
+[TRAIN]
+# lr_scheduling patience and early_stopping_thresh are based on the performance_tracking_interval
+learning_rate = 0.0001
+batch_size = 10
+max_epoch = 100
+num_worker = 4
+use_lr_scheduler = False
+lr_scheduler_patience = 3
+early_stopping_thresh = 7
+
+[INFERENCE]
+# Fill this part after the training
+# Check wandb_id on the wandb platform according to experiment
+target_data_folder = /path/to/target/folder
+target_ann = /path/to/target/annotation/file
+wandb_id = wandb_unique_id
+model_checkpoint_path = /path/to/model/checkpoint
+ontology_filepath = /path/to/ontology.json
+confidence_threshold = 0.5
```

Comment:
This section is not really self-explanatory. Could we add some additional comments there?